### ISSUE TITLE
Co-author graph

### ISSUE THREAD

**Issue by TS404** [fnielsen/scholia#910]
It's always bugged me that each author pair in the Co-author graph has two double-headed arrows. Surely only a single double-headed arrow is needed as each edge. Is this achievable with wikidata queries?

**Comment by fnielsen** [comment]
Yes, that bothers me too. AFAIK the WDQS cannot do undirected links. So either one would need to modify the WDQS code or build an entirely new graph rendering.
